src/app/legacy/containers/CpsRecommendations/index.jsx

```diff
@@ -86,7 +90,11 @@ const CpsRecommendations = ({ items }) => {
   return (
     <ErrorBoundary recommendations={items}>
       <GridItemMediumNoMargin>
-        <RecommendationsWrapper data-e2e={labelId} {...a11yAttributes}>
+        <RecommendationsWrapper
+          data-e2e={labelId}
+          {...a11yAttributes}
+          isArticlePage={pageType === ARTICLE_PAGE}
```

Comment:
You can use your previously defined `isArticlePage` here

Reply:
Lint gives me an error 😔